blog/_posts/2017-09-26-banshee-bandits-wrap-up.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Banshee Bandits Wrap Up
+layout: post
+created_at: Tues Sept 26 2017
+permalink: blog/2017-09-26-banshee-bandits-wrap-up
+current: blog
+author: Shelby & Lillian
+twitter: bansheebandits
+categories:
+- student-posts
+---
+
+# It's the end of the line...time to wrap up!
+
+## What We've Learned
+
+* How to Read a Giant Website’s Repo
```

Comment:
Mark down's bullet point stylle doesn't work properly if you have related paragraph after that. Because it doesn't align that text in the same alignment of the bullet points. Can you fix it? [Hint: you can use header style instead]